packages/lib/src/prod/remote-production.ts

```diff
@@ -199,18 +199,9 @@ export function prodRemotePlugin(
       if (builderInfo.isRemote) {
         for (const expose of parsedOptions.prodExpose) {
           if (!expose[1].emitFile) {
-            if (!expose[1].id) {
-              // resolved the moduleId here for the reference somewhere else like #152
-              expose[1].id = (await this.resolve(expose[1].import))?.id
-              if (!expose[1].id) {
-                this.error(
```

Comment:
I thik this error tips should be kept. See this issue #153

Reply:
I tried to modify an incorrect exposure url. It seems that the prompt message of rollup can explain the problem.
<img width="462" alt="image" src="https://user-images.githubusercontent.com/70385062/227829491-db544eea-0971-46d8-92c5-eb823f5336b7.png">

<img width="663" alt="image" src="https://user-images.githubusercontent.com/70385062/227829336-c52e008c-f2da-4b9a-b743-74b80b3f5117.png">

Reply:
I also tried the rollup prompt, which is enough to explain the problem.
<img width="716" alt="image" src="https://user-images.githubusercontent.com/70385062/228105100-dedcb78d-bc02-41e4-a3f1-7d32bac2a478.png">